### PR TITLE
feat: add --select option to eval to control output

### LINKED ISSUE #195: Long output data of 'swio eval' with "input": and "output": keys.
When I run `swio eval -p . -i ./input -n seedwing::allow` from [seedwing-benchmarks/testcases/json/](https://github.com/seedwing-io/seedwing-benchmarks/tree/main/testcases/json). It reports ~14605 lines of output as below (excluding line numbers in the first column) when the evaluation is true.  I think the default option should be to not print the input and output data with the result.

```
    1 {
    2   "name": {
    3     "pattern": "seedwing::allow"
    4   },
   5   "input": {
    6     "bomFormat": "CycloneDX",
    7     "components": [
    8       {
    9         "bom-ref": "pkg:maven/io.quarkus/quarkus-arc@2.16.2.Final?type=jar",
  ...
 7300     "serialNumber": "urn:uuid:5dce5040-6d92-4eef-8b73-deb5cd8fad46",
 7301     "specVersion": "1.4",
 7302     "version": 1
 7303   }, 
 7304   "output": {
 7305     "bomFormat": "CycloneDX",
 7306     "components": [
 7307       {
 7308         "bom-ref": "pkg:maven/io.quarkus/quarkus-arc@2.16.2.Final?type=jar",
 ...
14599     "serialNumber": "urn:uuid:5dce5040-6d92-4eef-8b73-deb5cd8fad46",
14600     "specVersion": "1.4",
14601     "version": 1
14602   },
14603   "severity": "none",
14604   "reason": "Because all fields were satisfied"
14605 }
```

### PR DESCRIPTION
**Pull request by jcrossley3**
Default fields are "name,severity,reason,rationale"

Fixes #195

### CODE REVIEW

File: swio/README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 The Seedwing Policy CLI provides access to the functionality of the Seedwing
 Policy Engine without requiring access to a running Policy Server.
 
-See the [documentation](https://docs.seedwing.io/cli/dev/index.html) for examples.
+See the [documentation](https://docs.seedwing.io/cli/index.html) for examples.
```
Comment (lkatalin) [inline swio/README.md:6]: Is this a related change?

**Comment by lkatalin** [comment]
Thanks @jcrossley3 !

File: swio/README.md (hunk 1 above)
Comment (jcrossley3) [reply swio/README.md:6]: Well, I only found that bug trying to figure out how to use the cli in order to fix this issue, so... yes? :smile:

Comment (jcrossley3) [reply swio/README.md:6]: @lkatalin do you want me to remove this change from the PR before you approve it? I can go either way, but I'd like to have it merged before the next nightly build to unblock @niteeshkd

Comment (lkatalin) [reply swio/README.md:6]: Ah, not at all, I was just curious. The only reason I didn't approve is because I didn't have the time to test it (or Niteesh's error). You should go ahead and merge it.

**Review by lkatalin — APPROVED** [review]
